state_transfer/example_test.go

```diff
@@ -10,7 +10,8 @@ import (
 	"github.com/konveyor/crane-lib/state_transfer"
 	"github.com/konveyor/crane-lib/state_transfer/endpoint"
 	"github.com/konveyor/crane-lib/state_transfer/endpoint/route"
-	"github.com/konveyor/crane-lib/state_transfer/labels"
+	"github.com/konveyor/crane-lib/state_transfer/meta"
+	metadata "github.com/konveyor/crane-lib/state_transfer/meta"
```

Comment:
NIT: rename this to `statetransfermeta`? to avoid confusion with resource metadata?